db/compaction_iterator.cc

```diff
@@ -292,6 +302,18 @@ void CompactionIterator::NextFromInput() {
       ikey_.user_key = current_key_.GetUserKey();
     }
 
+    if (UNLIKELY(!current_key_committed_)) {
```

Comment:
Put a TODO to change this to LIKELY when our default txn policy changes.

Reply:
Will refactor a bit to make this cleaner.